tests/specs/controllers/OpenEAppController.test.js

```diff
@@ -148,4 +162,56 @@ describe('OpenEAppController', () => {
             );
         });
     });
+
+    describe('date countdown', () => {
+        beforeEach(async () => {
```

Comment:
`OpenEAppController. _calculateDaysLeftToDownload` is quite an important method here. I think there should be some unit tests to make sure it does return (or throw) what you're expecting.

Reply:
Yeah, this is a good point. Some of my other tests sort of test this function already since this needs to return the right value for the others to pass but I'm happy to add specific tests for this function as well.